go/cron/pfflow.go

```diff
@@ -0,0 +1,158 @@
+package maint
+
+import (
+	"net/netip"
+	"time"
+)
+
+//easyjson:json
```

Comment:
Remove

Reply:
This is needed for code generation with easyjson